src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs

```diff
@@ -394,11 +391,11 @@ public int Count
     // The operations exposed on the BufferManager class are not thread safe. 
     internal class BufferManager
     {
-        int _numBytes;  // The total number of bytes controlled by the buffer pool.
-        byte[] _buffer; // The underlying byte array maintained by the Buffer Manager.
-        Stack<int> _freeIndexPool;
-        int _currentIndex;
-        int _bufferSize;
+        private int _numBytes;  // The total number of bytes controlled by the buffer pool.
+        private byte[] _buffer; // The underlying byte array maintained by the Buffer Manager.
+        private Stack<int> _freeIndexPool;
+        private int _currentIndex;
+        private int _bufferSize;
```

Comment:
Looks like _bufferSize, _freeIndexPool, _buffer, and _numBytes could all be made readonly.